docs/core/deploying/runtime-package-store.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Runtime package store
+description: Runtime package store and target manifests
+keywords: .NET, .NET Core
+author: beleroy
+ms.date: 04/13/2017
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+---
+
+# Runtime package store
+
+Starting with .NET Core 2.0, it is possible to package and deploy applications against a known set of packages that is known to exist on the target environment. The benefits in doing so is that it is possible to deploy smaller published applications, to use less disk space, and in some cases to improve startup performance.
+
+This feature is implemented by a runtime package store, which is a location on disk where packages are stored, that the runtime can find, access, and use. That location is a `store` directory, next to the `dotnet` host. Under this directory, there are subdirectories for target frameworks, under which the package store follows a NuGet layout.
```

Comment:
we style directory names with italic

should you link to the dotnet topic and the target frameworks topic?